yala/linters.py

```diff
@@ -173,5 +173,30 @@ def parse(self, stdout_lines, stderr_lines):
         return self._parse_by_pattern(stdout_lines, pattern), stderr_lines
 
 
+class Black(Linter):
+    """Parser for black code formatter lint check."""
+
+    name = 'black'
+    command = 'black --check'
+
+    def parse(self, stdout_lines, stderr_lines):
+        """Parse linter stdout and stderr lines.
+
+        Expected error message:
+
+        would reformat file1.py
+        would reformat file2.py
+        Oh no! 💥 💔 💥
+        2 files would be reformatted.
+        """
+        pattern = re.compile(r'''^.*?would\sreformat\s(?P<path>.+\.py)$''',
+                             re.VERBOSE | re.IGNORECASE)
+        return self._parse_by_pattern(stderr_lines, pattern), stdout_lines
+
+    def _create_output_from_match(self, match_result):
```

Comment:
Have you tried without overriding this method? Maybe you can safely delete it.

Reply:
Indeed. I managed to get rid of it, I parametrized the `?P<msg>` in regex.

Reply:
> Than you! Yes, let's add it! Let's just be careful not to break current linted source codes. Personally, I prefer changing Black options than isort, pycodestyle and others, but that's something each user should be able to decide.

Awesome. Thanks, @cemsbr, I've pushed some commits based on your review, feel free to also push or squash any commit if needed. Appreciated your help, exciting to `black` on `yala`.